### PR TITLE
[SHELL32] Polish translation for "Problem with Shortcut" dialog

### PR DESCRIPTION
**Pull request by pithwz**
pl-pl Polish translation update for new dialog with SHORTCUT

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-polish

File: dll/win32/shell32/lang/pl-PL.rc [hunk 1]
```diff
@@ -1003,5 +1003,5 @@ BEGIN
     IDS_EDITING_ACTION "Edytowanie akcji dla typu: "
     IDS_NO_ICONS "Plik '%s' nie zawiera ikon.\n\nWybierz ikonę z listy lub określ inny plik."
     IDS_FILE_NOT_FOUND "Nie odnaleziono pliku '%s'."
-    IDS_LINK_INVALID "The item '%s' that this shortcut refers to has been changed or moved, so this shortcut will no longer work properly."
+    IDS_LINK_INVALID "Element '%s', , do którego odnosi się ten skrót, został zmieniony lub przesunięty, więc ten skrót nie będzie już działał poprawnie."
```
Comment (Saibamen) [inline dll/win32/shell32/lang/pl-PL.rc:1006]:
```suggestion
    IDS_LINK_INVALID "Element '%s', do którego odwołuje się ten skrót, został zmieniony lub przeniesiony i dlatego skrót ten nie będzie działał poprawnie."
```

File: dll/win32/shell32/lang/pl-PL.rc [hunk 2]
```diff
@@ -716,17 +716,17 @@ BEGIN
...
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON 0, IDC_LINK_PROBLEM_ICON, 5, 5, 0, 0
-    LTEXT "The item '%s' that this shortcut refers to has been changed or moved, so this shortcut will no longer work properly.", IDC_LINK_PROBLEM_LABEL1, 35, 5, 210, 35
-    LTEXT "Nearest match based on name, type, size, and date:", IDC_LINK_PROBLEM_LABEL2, 35, 45, 210, 35
-    LTEXT "Do you want to fix this shortcut to point to this target or do you just want to delete it?", -1, 35, 85, 210, 30
-    DEFPUSHBUTTON "&Fix it", IDOK, 55, 120, 60, 15, WS_DISABLED
-    PUSHBUTTON "&Delete it", IDYES, 120, 120, 60, 15
-    PUSHBUTTON "Cancel", IDCANCEL, 185, 120, 60, 15
+    LTEXT "Element  '%s', do którego odwołuje się ten skrót, został zmieniony lub przeniesiony i dlatego skrót ten nie będzie działał poprawnie.", IDC_LINK_PROBLEM_LABEL1, 35, 5, 210, 35
```
Comment (Saibamen) [inline dll/win32/shell32/lang/pl-PL.rc:724]:
```suggestion
    LTEXT "Element '%s', do którego odwołuje się ten skrót, został zmieniony lub przeniesiony i dlatego skrót ten nie będzie działał poprawnie.", IDC_LINK_PROBLEM_LABEL1, 35, 5, 210, 35
```

File: dll/win32/shell32/lang/pl-PL.rc [hunk 3]
```diff
@@ -716,17 +716,17 @@ BEGIN
...
+    LTEXT "Najbliższy odpowiednik pod względem rozmiaru, daty i typu:", IDC_LINK_PROBLEM_LABEL2, 35, 45, 210, 35
```
Comment (Saibamen) [inline dll/win32/shell32/lang/pl-PL.rc:725]:
```suggestion
    LTEXT "Najbliższy odpowiednik pod względem nazwy, typu, rozmiaru i daty:", IDC_LINK_PROBLEM_LABEL2, 35, 45, 210, 35
```

**Comment by katahiromz** [comment]
Can you provide a screenshot or a resource design image?

**Comment by pithwz** [comment]
fixed_shortcut_DLG_PLpl

**Pull request merged**